teams/teams-ps/teams/Update-M365TeamsApp.md

```diff
@@ -33,11 +33,25 @@ This cmdlet allows administrators to modify app state and availability by adding
 ### Example 1
 
 ```powershell
-PS C:\> Update-M365TeamsApp -Id 3acac8f6-4e9e-4ee1-9c0e-0ea837d15ea3 -IsBlocked $true -AppAssignmentType UsersAndGroups -OperationType Add -Users eec823bd-0979-4cf8-9924-85bb6ffcb57d -Groups 37da2d58-fc14-453e-9a14-5065ebd63a1d
+PS C:\> Update-M365TeamsApp -Id 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b -AppAssignmentType Everyone
 ```
+Updates availablity assignment information of 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b app to Everyone.
```

Comment:
Updates the availability value for Bookings app (App ID 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b) to Everyone.

Reply:
Updated as requested